tests/wasmfs/wasmfs_mkdir.c

```diff
@@ -61,8 +61,11 @@ int main() {
   // Try to make a directory with an empty pathname.
   errno = 0;
   mkdir("", 0777);
-  printf("Errno: %s\n", strerror(errno));
+#ifdef WASMFS
```

Comment:
Please add a comment explaining why these diverge, and a TODO if we intend to fix that.